### PR TITLE
Implement unit tests for the network utility code

### PR DESCRIPTION
**Pull request by timvandermeij**
This should provide 100% coverage for the file.

### CODE REVIEW

File: test/unit/network_utils_spec.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  describe('validateRangeRequestCapabilities', function() {
+    const defaultValues = {
+      allowRangeRequests: false,
+      suggestedLength: undefined,
+    };
+
+    it('rejects range chunk sizes that are not strictly positive', function() {
+      expect(function() {
+        validateRangeRequestCapabilities({ rangeChunkSize: 0, });
+      }).toThrow(new Error());
```
Comment (Snuffleupagus) [inline test/unit/network_utils_spec.js]: This is an existing issue, but can you please add a suitable error message to the `assert` in the core code, such that we can test that the "correct" error is thrown here?

**Review by Snuffleupagus — APPROVED** [review]
Looks good to me, with one small question, thank you!

*Edit:* Merry Christmas!

File: test/unit/network_utils_spec.js (hunk 1 above)
Comment (timvandermeij) [reply test/unit/network_utils_spec.js]: Done in the new commit.

Thank you for the review, and I wish you a merry Christmas too!

**Pull request merged**